pkg/knode-manager/knode.go

```diff
@@ -53,35 +61,35 @@ func NewKnode(ctx context.Context, knode *kosmosv1alpha1.Knode, cmdConfig *confi
 		return nil, fmt.Errorf("could not build clientset for worker cluster %s: %v", knode.Name, err)
 	}
 
-	informer := kubeinformers.NewSharedInformerFactory(client, 0)
-	podInformer := informer.Core().V1().Pods()
-	nsInformer := informer.Core().V1().Namespaces()
-	nodeInformer := informer.Core().V1().Nodes()
-	cmInformer := informer.Core().V1().ConfigMaps()
-	secretInformer := informer.Core().V1().Secrets()
-	serviceInformer := informer.Core().V1().Services()
+	cInformer := kubeinformers.NewSharedInformerFactory(client, 0)
+	cPodInformer := cInformer.Core().V1().Pods()
+	cNsInformer := cInformer.Core().V1().Namespaces()
+	cNodeInformer := cInformer.Core().V1().Nodes()
+	cCmInformer := cInformer.Core().V1().ConfigMaps()
+	cSecretInformer := cInformer.Core().V1().Secrets()
+	cServiceInformer := cInformer.Core().V1().Services()
```

Comment:
what's meaning of c?

Reply:
cxxx => client  : the client for member cluster
mxx => master: the client for host cluster

Reply:
updated